src/main/java/microsoft/exchange/webservices/data/HttpWebRequest.java

```diff
@@ -220,18 +213,14 @@ public void setUrl(URL url) {
   }
 
   /**
-   * Checks if is pre authenticate.
-   *
-   * @return true, if is pre authenticate
+   * Whether to use preemptive authentication. Currently not implemented, though.
```

Comment:
Any benefit of having this while it doesn't have an effect?  I can see we have other public interface changes here, so it is not like we don't want to change the public interface.